pom.xml

```diff
@@ -193,7 +193,7 @@
     <slf4j.version>1.7.36</slf4j.version>
     <soebes.itf.version>0.11.0</soebes.itf.version>
     <spark3.jackson.version>2.13.1</spark3.jackson.version>
-    <spark3.version>3.1.2</spark3.version>
+    <spark3.version>3.2.1</spark3.version>
```

Comment:
just for my understanding:
why can we update this here and need to update all the docs to the spark version specific runtime... but we ourselves still use the "generic" spark runtime artifact from iceberg in our build
https://github.com/projectnessie/nessie/blob/c8b2becbb52e93381c320295c7b936fca6eeaf3d/pom.xml#L362-L365
?